hugr-core/src/builder.rs

```diff
@@ -26,11 +26,11 @@
 //! `CircuitBuilder`.
 //!
 //! ```rust
-//! # use hugr::Hugr;
-//! # use hugr::builder::{BuildError, BuildHandle, Container, DFGBuilder, Dataflow, DataflowHugr, ModuleBuilder, DataflowSubContainer, HugrBuilder};
-//! use hugr::extension::prelude::BOOL_T;
-//! use hugr::std_extensions::logic::{NotOp, LOGIC_REG};
-//! use hugr::types::FunctionType;
+//! # use hugr_core::Hugr;
+//! # use hugr_core::builder::{BuildError, BuildHandle, Container, DFGBuilder, Dataflow, DataflowHugr, ModuleBuilder, DataflowSubContainer, HugrBuilder};
+//! use hugr_core::extension::prelude::BOOL_T;
```

Comment:
should these have `#` in front of them

Reply:
I missed module-level docs when searching for examples...

I think these give a bit of context to the example below, so it's fine to have them explicit.
I changed all `hugr_core::` uses for `hugr::`, to avoid further confusions.